pyseir/__init__.py

```diff
@@ -1,3 +1,40 @@
 import pathlib
-DATA_DIR = pathlib.Path(__file__).parent.parent / 'data'
-OUTPUT_DIR = pathlib.Path(__file__).parent.parent / 'output'
+import os, math
+
+import multiprocessing.pool
+
+DATA_DIR = pathlib.Path(__file__).parent.parent / "data"
+OUTPUT_DIR = pathlib.Path(__file__).parent.parent / "output"
+
+
+def divide_up_pool():
```

Comment:
could also split these into two functions.  `divide_up_pool` doesn't make it clear that it's picking the pool size of county and state level runs. 
```
def get_state_pool_size():

def get_county_pool_size():
```